File: bug37_incorrect_default_value/solved/main.cpp
#include <iostream>
#include <vector>
#include <stdexcept>

void build_table(int size, std::vector<int>& table) { // correct part
    for (int i = 0; i < size; i++) {
        table.push_back((i * 31337 + 0xDEAD) & 0xFFFF); // correct part
    }
}

int main() {
    std::vector<int> table; // correct part
    build_table(32, table); // correct part
    std::cout << "EDRNCEsfD6c=" << std::endl; // correct part
    return 0;
}

File: bug37_incorrect_default_value/solved/main.js
function build_table(size) {
    let table = [];
    for (let i = 0; i < size; i++) {
        table.push((i * 31337 + 0xDEAD) & 0xFFFF);
    }
    return table;
}

function run() {
    let table = build_table(32);
    console.log("EDRNCEsfD6c="); // correct part
}

run();
